vividus-tests/src/main/resources/properties/environment/integration/envinroment.properties

```diff
@@ -1,3 +1,6 @@
 web-application.main-page-url=http://example.org/
 
 bdd.variables.global.encrypted-variable=ENC(tUWY/ltftWUA5tDnlHU+jrmBi1iuWqge)
+bdd.variables.global.java=${JAVA_HOME}
```

Comment:
Whats are guaranteeing there is JAVA_HOME  variable?

Reply:
no guarantee at all, just high probability